support-frontend/app/views/contributions.scala.html

```diff
@@ -36,12 +39,20 @@
       @user.publicFields.displayName.map { displayName =>
         displayName: "@displayName",
       }
-    @for(fields <- user.privateFields; firstName <- fields.firstName; lastName <- fields.secondName) {
-        firstName: "@firstName",
-        lastName: "@lastName",
+      @for(fields <- user.privateFields) {
+        @for(firstName <- fields.firstName; lastName <- fields.secondName) {
+          firstName: "@firstName",
+          lastName: "@lastName",
+        }
+        @for(address4 <- fields.billingAddress4 orElse fields.address4) {
```

Comment:
We need to add extra privileges to our API key to retrieve this value properly (certainly in DEV/CODE).